pycbc/waveform/bank.py

```diff
@@ -567,14 +572,14 @@ def __getitem__(self, index):
         buff_size = pycbc.waveform.get_waveform_filter_length_in_time(approximant, **p)
         
         tlen = self.round_up((buff_size + min_buffer) * self.sample_rate)
-        flen = tlen / 2 + 1
+        flen = int(tlen / 2 + 1)
 
         delta_f = self.sample_rate / float(tlen)
 
         if f_end is None or f_end >= (flen * delta_f):
             f_end = (flen-1) * delta_f
 
-        logging.info("Generating %s, %ss, %i" % (approximant, 1.0/delta_f, index))
+        logging.info("Generating %s, %ss, %i, starting from %s Hz" % (approximant, 1.0/delta_f, index, flow))
```

Comment:
You can avoid the % operator here and just pass the variables to logging.info(),
```
logging.info("Generating %s, %ss, %i, starting from %s Hz", approximant, 1.0/delta_f, index, flow)
```